scripts/rotating_camera/controllers/rotating_camera/rotating_camera.c

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 1996-2020 Cyberbotics Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <math.h>
+#include <webots/robot.h>
+#include <webots/supervisor.h>
+
+#define TOL 1e-6
+#define ANGLE_REV 6.28
+#define CW  0
+#define CCW 1
+#define UP 0
+#define DOWN 1
+
+// Global variables
+WbNodeRef root, nodes;
+WbNodeRef nodeViewPoint;
+WbNodeRef nodeRobot;
+
+WbFieldRef children;
+WbFieldRef fieldViewPointPos;
+WbFieldRef fieldViewPointOrient;
+WbFieldRef fieldRobotTranslation;
```

Comment:
We should remove the `field` prefix, fix `Viewpoint` spelling, not use abbreviations (e.g., Pos &rarr; position) and respect our C coding style with respect to case and underscores: `fieldRobotTranslation` &rarr; `robot_translation`;